PowerEditor/src/NppBigSwitch.cpp

```diff
@@ -115,6 +116,8 @@ LRESULT Notepad_plus_Window::runProc(HWND hwnd, UINT message, WPARAM wParam, LPA
 						SWP_FRAMECHANGED);
 				}
 
+				::SendMessage(hwnd, NPPM_INTERNAL_HANDLE_OS_APP_RESTART, (WPARAM)false, 0);

```

Comment:
Why do we need `NPPM_INTERNAL_HANDLE_OS_APP_RESTART`?
Why don't we call `SetOSAppRestart(bool bLog)` directely?

Reply:
Again, this is a remnant from the previous PR version, now it is not needed. I will simplify that.